### ISSUE TITLE
Output files

### ISSUE THREAD

**Issue by Geetima2021** [Sud-ana/tcl_scripting#3]
The output file is important as it can help us to check the whether the outcome of your program matches the one you attached in your repository. It also gives a picture of all the files generated which can be helpful for a person leaning TCL via your repository.

**Comment by Sud-ana** [comment]
> The output file is important as it can help us to check the whether the outcome of your program matches the one you attached in your repository. It also gives a picture of all the files generated which can be helpful for a person leaning TCL via your repository.

Of course. The output files and the full repository is uploaded in assets.
